External/SPMmodified/xASL/xASL_bids_csv2tsvReadWrite.m

```diff
@@ -45,7 +45,7 @@
 
 [Fpath, Ffile, Fext] = xASL_fileparts(PathIn);
 
-if isempty(regexp(Fext, '^\.(tsv|csv)$'))
+if isempty(regexp(Fext, '^\.(tsv|csv)$', 'once'))
```

Comment:
You keep adding 'once' because Matlab complaints, I would argue this just makes stuff less readable, we won't notice the speed increase ;)

Reply:
The speed is not important - but it removes Matlab warnings - which is nice to have. Not to optimize code, but to have code without warnings. But we can for sure skip this the next time...